pkgs/games/space-station-14-launcher/space-station-14-launcher.nix

```diff
@@ -56,7 +56,12 @@ buildDotnetModule rec {
   };
 
   # SDK 6.0 required for Robust.LoaderApi
-  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_8_0 sdk_6_0 ];
+  dotnet-sdk =
+    with dotnetCorePackages;
+    sdk_8_0
+    // {
+      inherit (sdk_8_0) packages targetPackages;
+    };
```

Comment:
Is this really needed here? it's just combining 8.0 with 8.0.

Reply:
Yeah, that's a mistake.  I must have switched it from 6 -> 8.

Reply:
(coming from https://github.com/NixOS/nixpkgs/issues/326335#issuecomment-2521197344) If I understand correctly, you mistakenly bumped the inner `sdk_6_0` to `sdk_8_0` and you didn't intend to fix the EOL warning?
However if you did intend to fix or work around the EOL warning, can you explain how it doesn't actually remove the older SDK from the package?